torch/distributed/_tensor/op_schema.py

```diff
@@ -129,15 +129,16 @@ class TupleStrategy(StrategyType):
     TupleStrategy represents the output strategy of this op is a tuple
     of strategy, i.e. If the output of this op is a tuple of tensors or list of tensors
     with possibly different placement strategies, we should return a TupleStrategy that
-    contains a tuple of OpStrategy.
+    contains a tuple of OpStrategy. For None return value, the corresponding OpStrategy
+    object in childs should also be None.
 
     NOTE: if the output of the op is a List[Tensor] and they share the same placement
     strategy, then we should return a single OpStrategy instead of a TupleStrategy
     """
 
-    def __init__(self, childs: Sequence[StrategyType]) -> None:
+    def __init__(self, childs: Sequence[Optional[StrategyType]]) -> None:
```

Comment:
did you try to run lintrunner locally and whether it passes the lint checks? I was expecting we have bunch of callsites we need to add asserts to satisfy mypy

Reply:
Yes I have run local lintrunner and it reports no issue. 
1) I also expected to add asserts but found there're already existing assertions checking types. So there is no need.
2) I checked the lint error in CI and it's unrelated. But obviously here the length issue was not captured by the CI linter so there could be something going wrong.